CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing
+
+This document is meant to better describe the architecture of this
+project, thus making it easier for other developers to jump in
+and understand how this piece of software works.
+
+## The server
```

Comment:
It would also be good if you describe the file structure of the whole IPM project, not just the server. That his, how different files etc. are related to each other.

Reply:
I did add the structure of the project in the architecture.md file. I did add a tree-like display of how files are organized, also explaining them in the architecture.md. Let me know if you like the tree-like display of the hierarchy of the project or if I should remove it

Reply:
I did not mean a file tree, but rather that you give an overview and describe the key files and how they are related.

Reply:
Removed the file tree